pom.xml

```diff
@@ -0,0 +1,230 @@
+        <timestamp>${maven.build.timestamp}</timestamp>
+        <maven.build.timestamp.format>dd-MMMM-yyyy HH:mm:ss</maven.build.timestamp.format>
+    </properties>
+
+    <repositories>
+        <!-- Let's first add the default Maven central repo,
+             otherwise Maven will not use Maven Central as first download option -->
+        <repository>
+            <id>central</id>
+            <name>Maven Repository Switchboard</name>
+            <layout>default</layout>
+            <url>https://repo1.maven.org/maven2</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+        <repository>
+            <id>maven-eu</id>
+            <name>maven-eu Repository</name>
+            <url>https://maven-eu.nuxeo.org/nexus/content/repositories/public/</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+        <repository>
+            <id>slubnexus-public</id>
+            <name>Slub-Nexus:3rd Party Releases</name>
+            <url>http://nexus.slub-dresden.de:8081/nexus/content/repositories/public/</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+        </repository>
+        <repository>
+            <id>slubnexus-snapshots</id>
+            <name>Slub-Nexus:Internal Snapshots</name>
+            <url>http://nexus.slub-dresden.de:8081/nexus/content/repositories/snapshots/</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+        <repository>
+            <id>slubnexus-releases</id>
+            <name>Slub-Nexus:Internal Releases</name>
+            <url>http://nexus.slub-dresden.de:8081/nexus/content/repositories/releases/</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.lowagie</groupId>
+            <artifactId>itext</artifactId>
+            <version>2.1.5</version>
+        </dependency>
+        <dependency>
+            <groupId>com.thoughtworks.xstream</groupId>
+            <artifactId>xstream</artifactId>
+            <version>1.3.1</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
+            <version>1.2</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-configuration</groupId>
+            <artifactId>commons-configuration</artifactId>
+            <version>1.6</version>
+            <exclusions>
+                <exclusion>
+                    <artifactId>commons-logging</artifactId>
+                    <groupId>commons-logging</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
```

Comment:
I would advice commons-io 2.5 - 1.4 introduce problems to kitodo-production (FileManagement module has conflicts).